content/blog/2025-03-11-Alive-Peephole-Optimizations

```diff
@@ -0,0 +1,62 @@
++++
+title = "Alive: Provably Correct Peephole Optimizations"
+[extra]
+[[extra.authors]]
+name = "Annabel Baniak"
+[[extra.authors]]
+name = "Katherine Wu"
+[[extra.authors]]
+name = "Max Fan"
+[[extra.authors]]
+name = "Stephanie Ma"
++++
+
+# Alive: Provably Correct Peephole Optimizations
+A blog post about the paper ["Provably Correct Peephole Optimizations With Alive"](https://dl.acm.org/doi/pdf/10.1145/2737924.2737965).
```

Comment:
I don't think you need this sentence… you can just link to the paper from the first sentence of your overview.